### DISCUSSION TITLE
transform.Rotation버그

### DISCUSSION THREAD

**Discussion by Andwhy81kr** [naverz/zepeto-studio-kor#930] [5. Q&A]
this.transform.Rotate(Vector3.up, 0);

로 입력하면, angle값과 상관없이 계속 돌아갑니다.
Axis 값에따라 방향은 바뀌지만 angle값과 상관없이 돌고있는것으로 보이네요.
(에디터에서 발생합니다. 디바이스에선 테스트하지 않았습니다.)

World 1.3.0입니다.

**Comment by Andwhy81kr** [comment]
아..그리고 카테고리는 어떻게 선택하나요? 저한텐 3,4,5 만 노출되어 버그로 선택이 불가능하네요.

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 버그 리포트는 이슈 트래킹 및 로드맵 관리를 위해서 이슈게시판을 이용해 주시기를 부탁 드립니다 :)
https://github.com/naverz/zepeto-studio-kor/issues

기존의 버그 리포트 카테고리는 히스토리를 위해서 Read only로 관리 중입니다.
감사합니다.
